src/api/entities/CheckpointSchedule/__tests__/index.ts

```diff
@@ -154,6 +155,33 @@ describe('CheckpointSchedule class', () => {
     });
   });
 
+  describe('method: getCheckpoints', () => {
+    test('should return all the checkpoints created by the schedule', async () => {
+      const schedule = new CheckpointSchedule(
+        { id, ticker, start, period, remaining, nextCheckpointDate },
+        context
+      );
+      const firstId = new BigNumber(1);
+      const secondId = new BigNumber(2);
+      const rawFirstId = dsMockUtils.createMockU64(firstId.toNumber());
+      const rawSecondId = dsMockUtils.createMockU64(secondId.toNumber());
+
+      sinon.stub(utilsConversionModule, 'numberToU64');
```

Comment:
This one, on the other hand, should be stubbed to return the raw values when passed the non-raw ones